extension/src/setup/index.ts

```diff
@@ -504,14 +511,21 @@ export class Setup
       return undefined
     }
 
-    try {
-      return !(await this.internalCommands.executeCommand(
-        AvailableCommands.GIT_GET_REPOSITORY_ROOT,
-        cwd
-      ))
-    } catch {
+    return !(await this.internalCommands.executeCommand<string | undefined>(
+      AvailableCommands.GIT_GET_REPOSITORY_ROOT,
+      cwd
+    ))
+  }
+
+  private async getIsGitAccessible() {
+    if (this.gitAccessible) {
```

Comment:
[F] If it is installed then don't check again. If users want to uninstall Git mid-session then that is their problem.